reviewdog.go

```diff
@@ -15,16 +15,21 @@ import (
 // or linter, get diff and filter the results by diff, and report filterd
 // results.
 type Reviewdog struct {
-	p Parser
-	c CommentService
-	d DiffService
+	toolname string
+	p        Parser
+	c        CommentService
+	d        DiffService
 }
 
+var ExportedVarWithoutComment = 1 // TODO: revert
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var ExportedVarWithoutComment should have comment or be unexported